test/dialect/postgresql/test_reflection.py

```diff
@@ -1110,6 +1110,30 @@ def test_index_reflection_with_access_method(self):
                 "gin",
             )
 
+    @testing.fails_if("postgresql < 11.0", "indnkeyatts not supported")
+    @testing.provide_metadata
+    def test_index_reflection_with_include(self):
+        """reflect indexes with include set"""
+
+        metadata = self.metadata
+
+        Table(
+            "t",
+            metadata,
+            Column("id", Integer, primary_key=True),
+            Column("x", ARRAY(Integer)),
+            Column("name", String(20)),
+        )
+        metadata.create_all()
+        with testing.db.connect().execution_options(autocommit=True) as conn:
+            conn.execute("CREATE INDEX idx1 ON t (x) INCLUDE (name)")
+
+            ind = testing.db.dialect.get_indexes(conn, "t", None)
+            eq_(
+                ind,
+                [{"unique": False, "column_names": ["x"], "name": "idx1"}],
```

Comment:
this is not actually what that INDEX says, the INCLUDE information would need to be part of this.

Reply:
Currently SQLALchemy doesn't support INCLUDE indexes therefore this test checks reflection of such kind of index.